packages/@angular/cli/bin/ng

```diff
@@ -132,9 +133,20 @@ resolve('@angular/cli', { basedir: process.cwd() },
       cli = cli['default'];
     }
 
+    let standardInput;
+    if (process.platform === 'win32') {
```

Comment:
You are correct, this is residual from a different approach I had to start with. I'll remove the incorrect check for the platform, thanks.